app/src/main/res/values-xxhdpi/dimens.xml

```diff
@@ -5,4 +5,7 @@
   <dimen name="profile_chooser_fragment_profile_recycler_view_margin_top">36dp</dimen>
   <dimen name="profile_chooser_profile_view_view_margin_top">60dp</dimen>
   <dimen name="profile_chooser_add_view_margin_top_profile_not_added">60dp</dimen>
+
+  <!-- TopicInfoFragment -->
+  <dimen name="topic_info_fragment_topic_name_size">21sp</dimen>
```

Comment:
As I mentioned earlier we cannot change the value of text-size as per our needs. It should always be 24sp and also this solution will work for a specific screen only and on other screens it will fail.

Suggest look into these attributes for fixing this issue:

layout_width
constraints like start/end
gravity
Layout_gravity
Etc

Reply:
Okay actually I noticed few things
1. I  can solve this by margin end but for that I have to remove horizontal bias with that the marginEnd is not working.

Okay I'll see others options.